frontend/src/scenes/insights/ActionFilter/ActionFilterDropdown.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { ActionType } from '~/types'
+import { EventUsageType } from '~/types'
+import { EntityTypes } from '../trendsLogic'
+import { userLogic } from 'scenes/userLogic'
+import { actionsModel } from '~/models/actionsModel'
+import { FireOutlined, InfoCircleOutlined, AimOutlined, ContainerOutlined } from '@ant-design/icons'
+import './ActionFilterDropdown.scss'
+import { Tooltip } from 'antd'
+import { ActionSelectInfo } from '../ActionSelectInfo'
+import { entityFilterLogicType } from 'types/scenes/insights/ActionFilter/entityFilterLogicType'
+import { SelectBox } from '../../../lib/components/SelectBox'
+
+const getSuggestions = (events: EventUsageType[]): EventUsageType[] => {
+    return events
+        .filter((event) => event.usage_count > 0)
```

Comment:
Q: Should we even return these to the frontend?

Reply:
Do you mean the usage_count?

Reply:
No, events with usage_count == 0